model/src/main/java/com/plhal/ares/dblayer/DataRepository.java

```diff
@@ -1,4 +1,4 @@
-package com.plhal.ares.model;
+package com.plhal.ares.dblayer;
```

Comment:
Tady pouzivate balik `dblayer`, ale pritom nejde do modul `ares-api.dbLayer`. To zpusobi nekolik problemu. A je otazkou, zda v modulu `model` chcete tuto tridu nechat. Ono je to cele zpusobene tim, ze jste se rozhodl datove modely vracet pres "service" vrstvu. V takovem pripade se zda, ze modul `model` zustane prazdny. Co si o tom myslite?

Reply:
Tady ma byt package com.plhal.ares.model - to se mi nejak pri upravach podarilo prepsat. Ono mi celkove dava zabrat logicke cleneni projektu (do modulu, trid - kde jsou jednotlive kompetence kazde tridy, co patri kam atd.). Nevite o necem, kde je tohle popsano, pripadne nejaky projekt, kde by se to dalo "okoukat"?

Reply:
Ano, teorie je na internetu spousty, ale bohuzel se domnivam, ze je to jedna z veci, ktere se nejlepe clovek nauci praxi, jakmile zacne premyslet "objektove", v "komponentach" a z pohledu IOC. Nicmene verim, ze se nam to podari zvladnout i v tomto projektu.